### ISSUE TITLE
I'm not good at programming. I hope you can give me some guidance

### ISSUE THREAD

**Issue by chowlay1996** [OdysseasKr/neural-disaggregator#35]
Hello!
Thank you for your contribution. I would like to know how to add loss visualization function in RNN network. I would like to observe its fitting

**Comment by OdysseasKr** [comment]
Hello!

Since the models are built using Keras, you can get easily get the loss value per epoch. The `model.fit` function returns a History object containing all losses. You can then plot them with a few modifications in the code.

You can take a look at this blog post from [Machine Learing Mastery](https://machinelearningmastery.com/display-deep-learning-model-training-history-in-keras/) that explains the whole process.

**Comment by chowlay1996** [comment]
Thank you very much

**Comment by chowlay1996** [comment]
I have learned this blog, but model.fit () requires numpy Array or List for data types of X and y. Train_mains, train_meter in your code does not belong to this type. How can I transform the type

**Comment by OdysseasKr** [comment]
In order to work with the Keras function, you need to modify the RNN disaggregator, not just the code that calls it.

Take a look at https://github.com/OdysseasKr/neural-disaggregator/blob/master/RNN/rnndisaggregator.py#L95 for the RNN.

This line 
```
self.model.fit(mainchunk, meterchunk, epochs=epochs, batch_size=batch_size, shuffle=True)
```
receives two numpy arrays (`mainchunk` and `meterchunk`). This function returns the history object that you can use.

**Comment by chowlay1996** [comment]
I'd appreciate you for your help
